server/db/methods/StretchAnswer.js

```diff
@@ -1,37 +1,90 @@
 const { Op } = require('sequelize')
-const { User, CohortStretch, CohortUser, StretchAnswer } = require('../models')
+const {
+  User,
+  CohortStretch,
+  CohortUser,
+  StretchAnswer,
+  Cohort
+} = require('../models')
 
-const getAnswersOfStudentsOfSingleAdmin = function(adminId) {
+// These are the main parameters that are used in Sequelize models' find methods.
+const defaults = {
+  include: [
+    {
+      model: CohortStretch,
+      attributes: ['cohortId', 'stretchId'],
+      include: [
+        {
+          model: Cohort,
+          attributes: ['name']
+        }
+      ]
+    }
+  ]
+}
+
+// This method formats a single instance of StretchAnswer.
+StretchAnswer.prototype.format = function() {
+  const { cohortstretch, ...stretchAnswerFields } = this.dataValues
+  const { cohort, ...cohortsStretchFields } = cohortstretch.get()
+  return {
+    ...stretchAnswerFields,
+    ...cohortsStretchFields,
+    cohortName: cohort.name
+  }
+}
+
+StretchAnswer.prototype.addAssociations = function(format = true) {
+  return StretchAnswer.findByPk(this.id, defaults).then(sa =>
+    format ? sa.format() : sa
+  )
+}
+
+StretchAnswer.getAnswersOfStudent = function(studentId) {
+  return this.findAll({
+    where: { userId: studentId },
+    ...defaults
+  }).then(stretchAnswers => {
+    return stretchAnswers.map(sa => sa.format())
+  })
+}
+
+StretchAnswer.getAnswersOfStudentsOfSingleAdmin = function(adminId) {
   return User.findOne({
     where: { id: adminId },
     include: CohortUser
   })
     .then(user => user.cohortusers.map(cu => cu.cohortId))
     .then(cohortIds => {
-      console.log(cohortIds)
       return StretchAnswer.findAll({
         include: [
           {
             model: CohortStretch,
             attributes: ['cohortId'],
-            where: { cohortId: { [Op.in]: cohortIds } }
+            where: { cohortId: { [Op.in]: cohortIds } },
+            include: [
+              {
+                model: Cohort,
+                attributes: ['name']
+              }
+            ]
           }
         ]
       })
     })
     .then(stretchAnswers => {
```

Comment:
Would it be possible to replace this with:
```
  .then(stretchAnswers => stretchAnswers.map(s => s.format())
```
I am suggesting this because the return value for the current ``.then()`` appears slightly different to the return value of ``StretchAnswer.prototype.format()``.